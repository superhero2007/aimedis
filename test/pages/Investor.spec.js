import React from 'react';
import { shallow } from 'enzyme';

import Investor from 'pages/Investor';

function setup() {
  return shallow(<Investor />);
}

describe('Investor', () => {
  const wrapper = setup();

  it('should be a Component', () => {
    expect(wrapper.instance() instanceof React.Component).toBe(true);
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
