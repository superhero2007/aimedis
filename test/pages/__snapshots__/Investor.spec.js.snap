// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Investor should render properly 1`] = `"<div class=\\"app__investor app__route\\"><div class=\\"container\\"><section class=\\"app__investor__section1\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-7\\"><div class=\\"app__investor__section1__content\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><p class=\\"app__investor__section1__content__title\\">For Investors</p></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><p class=\\"app__investor__section1__content__subtitle\\">Executive Summary</p></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><p class=\\"app__investor__section1__content__description\\">Suspendisse interdum est at odio faucibus volutpat. Curabitur tincidunt ex non malesuada finibus. Etiam auctor, ante eu sollicitudin sodales, purus nibh venenatis dui, eu venenatis elit lorem quis purus. Donec sed tincidunt justo, sit amet pellentesque eros. In hac habitasse platea dictumst. Proin vitae dui vitae purus ultrices laoreet ultricies id lectus. In facilisis, enim eu aliquet fermentum, mauris nulla venenatis odio, ut tincidunt turpis dui eleifend nunc. Vivamus eleifend massa iaculis metus accumsan dapibus.</p></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><p class=\\"app__investor__section1__content__description\\">Vivamus semper placerat elit, vel dapibus nibh porttitor id. Etiam non velit risus. Ut quis erat vitae nulla accumsan fringilla et et neque. Interdum et malesuada fames ac ante ipsum primis in faucibus. In hac habitasse platea dictumst. Cras mollis augue ac massa suscipit blandit. Sed a nisl in ligula pulvinar tempor.</p></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><p class=\\"app__investor__section1__content__description\\">Proin et mollis erat. Morbi id lorem felis. Phasellus malesuada mauris eu convallis ornare. Quisque sit amet aliquam elit, eget ullamcorper magna. Sed elit augue, dapibus id congue quis, commodo sit amet arcu.</p></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__investor__section1__content__download\\"><button class=\\"btn btn__secondary\\" type=\\"button\\">DOWNLOAD THE CASE STUDY (8.4 MB)</button></div></div></div></div></div></section><section class=\\"app__investor__mask d-none d-lg-block\\"><div class=\\"app__investor__mask__bottom\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></section></div></div>"`;
